boards/adafruit/feather_esp32s3_tft/Kconfig.defconfig

```diff
@@ -0,0 +1,49 @@
+# Adafruit Feather ESP32S3 board configuration
+
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# Copyright (c) 2024 Leon Rinkel <leon@rinkel.me>
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_ADAFRUIT_FEATHER_ESP32S3_TFT_ESP32S3_PROCPU
+
+config HEAP_MEM_POOL_ADD_SIZE_BOARD
+	int
+	default 65535 if WIFI && BT
+	default 51200 if WIFI
+	default 40960 if BT
+	default 4096
```

Comment:
move to `Kconfig` and in Kconfig file have this board select `BOARD_LATE_INIT_HOOK`

Reply:
Same comment of previous board is applied to this file here.